releasenotes/notes/0.4/move-repo-c0b48ba3b0ced8db.yaml

```diff
@@ -4,6 +4,6 @@ other:
     The repository has been moved from 
     [github.com/Qiskit/qiskit-dynamics](https://github.com/Qiskit/qiskit-dynamics) to 
     [github.com/Qiskit-Extensions/qiskit-dynamics](https://github.com/Qiskit-Extensions/qiskit-dynamics), 
-    and the documentation has been moved from
-    [qiskit.org/documentation/dynamics](https://qiskit.org/documentation/dynamics) to 
-    [qiskit.org/ecosystem/dynamics](https://qiskit.org/ecosystem/dynamics/).
+    and the [documentation](https://qiskit-extensions.github.io/qiskit-dynamics/) has been moved from
+    qiskit.org/documentation/dynamics
+    qiskit.org/ecosystem/dynamics
```

Comment:
No point linking to the old sites since they redirect to the new location anyways.